<?xml version="1.0"?>

<launch>

    <!-- NODO QUE TRANSFORMA LA POSICIÓN DEL CENTROIDE DADA POR LA CÁMARA A LA REFERENCIA DEL MUNDO, ENVIÁNDOLA DESPUÉS AL NODO DE PREDICCIÓN -->
    <node name="pos_transformer" pkg="vision" type="pos_transformer" output="screen">
        <param name="platform_position_topic" value="/platform/estimated_position"/>
        <param name="world_estimated_position_topic" value="/world_estimated_position" />
        <param name="pub_rate" value="10" /> <!-- Hz -->
    </node>

   

</launch>
